### PR TITLE
Added Fastlane CI/CD tool link

### PR DESCRIPTION
**Pull request by vaimikpatel2908**
Open Source CI/CD for android and ios.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -504,6 +504,7 @@ Table of Contents
   * [codeship.com](https://codeship.com/) — 100 private builds/month, 5 private projects, unlimited for Open Source
   * [deployhq.com](https://www.deployhq.com/) — 1 project with 10 daily deployments (30 build minutes/month)
   * [drone](https://cloud.drone.io/) - Drone Cloud enables developers to run Continuous Delivery pipelines across multiple architectures - including x86 and Arm (both 32 bit and 64 bit) - all in one place
+  * [fastlane.tools](https://fastlane.tools/) - Fastlane is an open source platform aimed at simplifying Android and iOS deployment. Open Source CI/CD for development/release workflows.
```
Comment (ripienaar) [inline README.md:507]: Is this offered as a service with a free tier?

Comment (vaimikpatel2908) [reply README.md:507]: Yet it is totally free. Just need to set up workflows as usually we have to do for all CI/CD.

Comment (ripienaar) [reply README.md:507]: It does not look like a SaaS to me

Comment (vaimikpatel2908) [reply README.md:507]: When we use Codemagic or other CI/CD for continuous integration and continuous deployment we need to setup a yaml which keeps config for CI/CD. so, here we also do have to setup AppFile and Fastfile for CI/CD. But other platforms have limites build minutes or projects. This is totally free for unlimited projects/minutes.

Comment (vaimikpatel2908) [reply README.md:507]: But it is just for android and iOS applications

Comment (vaimikpatel2908) [reply README.md:507]: If you have any doubt checkout the Codemagic CI/CD set up/installation and compare it with fatlane. You will get it. Codemagic has two options: 1. Using web GUI  2. Using yaml file. Fastlane just provides second way and for initialization they have fatlane cli which helps for initial setup.

Comment (vaimikpatel2908) [reply README.md:507]: For locally it works with just cli command and distributes app automatically. Bu if you want to automate then just need to configure github workflows. Check out following links.
https://docs.flutter.dev/deployment/cd
https://docs.fastlane.tools/

Comment (vaimikpatel2908) [reply README.md:507]: Definitely. I would not say that it is a standalone service but definitely good alternative for the paid services.

Comment (ripienaar) [reply README.md:507]: Please review the purpose of the list. It’s for SaaS.

**Pull request closed without merging**